### PR TITLE
Release controller tools/imagestream relocation

### PR DESCRIPTION
**Pull request by bradmwilliams**
This PR scales the release-controllers down in preparation for moving the imagestreams and tools, from the api.ci cluster, to app.ci cluster.

This PR includes the changes from #14399

/cc @hongkailiu  @stevekuznetsov @jupierce

### CODE REVIEW

**Comment by bradmwilliams** [comment]
/hold
Holding until we are ready

File: hack/generators/release-controllers/content/osd_rc_deployments.py [hunk 1]
```diff
@@ -157,7 +157,7 @@ def _add_osd_rc_deployment(gendoc):
             'namespace': context.config.rc_deployment_namespace,
         },
         'spec': {
-            'replicas': 1,
```
Comment (hongkailiu) [inline hack/generators/release-controllers/content/osd_rc_deployments.py:160]: This is E4 in the doc.
I made a separate PR https://github.com/openshift/release/pull/14400
Easier to revert for E12 (scale up).

Comment (bradmwilliams) [reply hack/generators/release-controllers/content/osd_rc_deployments.py:160]: Right, but these changes must reflect the same state (replicas: 0) at that point in time or the release-controllers will all scale back up when these changes merge.

Comment (bradmwilliams) [reply hack/generators/release-controllers/content/osd_rc_deployments.py:160]: The alternative is to use this PR as the Scale Up step (E12).

Comment (hongkailiu) [reply hack/generators/release-controllers/content/osd_rc_deployments.py:160]: If scale down has to go with this PR.
Closing 14400.

**Pull request merged**